sumit.txt

```diff
@@ -1 +1,2 @@
-testing githuh
+practising how to use github commands on linux with VS code
```

Comment:
Looking good i see the changes happening :)
feel free to use this as practice ill upload a file in a moment for you so u can try and do a pull to get it